main.py

```diff
@@ -292,11 +295,11 @@ def generate_query(selector, ip_translator, mlab_site_resolver):
 
   server_ips = []
   try:
-    for retrieved_site_ip in mlab_site_resolver.get_site_ips(selector.site_name,
+    for retrieved_site_ip in mlab_site_resolver.get_site_ips(selector.site,
```

Comment:
This is a good idea. I filed an issue on myself in Observatory to get rid of the `_name` suffix there too:

https://github.com/opentechinstitute/mlab-observatory/issues/41